project_stage/models/project_stage.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# For copyright and license notices, see __manifest__.py file in module root
+# directory
+##############################################################################
+from odoo import models, fields
+
+
+class ProjectStage(models.Model):
+
+    _name = 'project.stage'
+    _description = 'Project Stage'
+    _order = 'sequence'
+
+    name = fields.Char(
+        string='Stage Name',
+        required=True,
+        translate=True,
+    )
+    description = fields.Text(
+    )
+    sequence = fields.Integer(
+    )
+    case_default = fields.Boolean(
+        'Default for New Projects',
+        help="If you check this field, this stage "
+             "will be proposed by default on each "
+             "new project. It will not assign this "
+             "stage to existing projects.",
+    )
+    fold = fields.Boolean(
```

Comment:
This field doesn't seem to actually fold the stage in the kanban view. Can you make sure it does?

Reply:
Could be that something is wrong with Kanban, because all of the empty stages do not show up in a kanban view. And I don't see why that is.